### PR TITLE
update to flutter 3.10.4

### PR DESCRIPTION
**Pull request by Danilosimoes**
update to flutter 3.10.4

### CODE REVIEW

File: analysis_options.yaml [hunk 1]
```diff
@@ -22,8 +22,11 @@ linter:
   # `// ignore_for_file: name_of_lint` syntax on the line or in the file
   # producing the lint.
   rules:
+    prefer_const_constructors: false
```
Comment (robsonoduarte) [inline analysis_options.yaml:25]: @Danilosimoes isn't  good idea disable rule like that, in my opinion the correct way to resolve this issue is fix in the code >> [here](https://dart.dev/tools/linter-rules#prefer_const_constructors) you can see the rule.

and you can use the tips os android studio:
image

**Review by robsonoduarte — CHANGES_REQUESTED** [review]
@Danilosimoes see my comment.

File: analysis_options.yaml (hunk 1 above)
Comment (Danilosimoes) [reply analysis_options.yaml:25]: Yes I agree, This is the correct way for resolve this problem. 
I'll resolve this problem with your solution and i learn a litlle bit.
Thanks

**Review by robsonoduarte — APPROVED** [review]
good job :muscle:

**Pull request merged**